docs-ref-conceptual/install-azure-cli-apt.md

```diff
@@ -35,10 +35,12 @@ for the Azure CLI. This package has been tested with:
    curl -L https://packages.microsoft.com/keys/microsoft.asc | sudo apt-key add -
    ```
 
-3. Install the CLI:
+3. Install the CLI and dependent libraries:
 
    ```bash
    sudo apt-get update
+   sudo apt-get install -y libssl-dev libffi-dev
```

Comment:
These libraries could all be on the same line as the other `install` command. Go ahead and put them all together, without the `-y` switch.

Reply:
Thanks. I've collapsed all the apt-get install into a single a single line